core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2923,6 +2923,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.NONE)
           .build();
+  public static final PropertyKey FUSE_SHELL_GROUP_MAPPING_ENABLED =
+      new Builder(Name.FUSE_SHELL_GROUP_MAPPING_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("Whether to map the user and group to the Unix user and group "
```

Comment:
Whether to translate Alluxio users and groups into Unix users and groups when exposing Alluxio files through the FUSE API. When this property is set to false, the user and group for all FUSE files will match the user who started the alluxio-fuse process.

Reply:
Thanks!! Changed!